assets/src/stories-editor/blocks/amp-story-page/edit.js

```diff
@@ -47,6 +52,10 @@ const TEMPLATE = [
 ];
 
 class EditPage extends Component {
+	shouldComponentUpdate() {
```

Comment:
Added the check for the CTA block to the Page block instead. Thought that perhaps it's better to just check the relevant page changes instead of going through all the blocks in the subscribe.